### PR TITLE
Oc 7941 CRF Version Metadata displaying duplicate items when two Layout Versions exist for CRF

### PR DESCRIPTION
**Pull request by jkeremian**

### CODE REVIEW

File: core/src/main/resources/properties/item_dao.xml [hunk 1]
```diff
@@ -112,7 +112,7 @@
 	</query>
 	<query>
 		<name>findAllItemsByVersionId</name>
-		<sql>select i.* from versioning_map v, item i 
+		<sql>select distinct i.* from versioning_map v, item i
```
Comment (yogi-oc) [inline core/src/main/resources/properties/item_dao.xml]: Do we need to add another table in the query? I am trying to understand why "distinct" is used.

File: web/src/main/java/org/akaza/openclinica/control/admin/CreateXformCRFVersionServlet.java [hunk 2]
```diff
@@ -41,6 +42,7 @@
     Locale locale;
     FileUploadHelper uploadHelper = new FileUploadHelper();
     public final String FM_BASEURL = "http://fm.openclinica.info:8080/api/protocol/";
+    // public final String FM_BASEURL = "http://oc.local:8090/api/protocol/";
```
Comment (yogi-oc) [inline web/src/main/java/org/akaza/openclinica/control/admin/CreateXformCRFVersionServlet.java]: you can remove this if not needed.

Comment (jkeremian) [inline web/src/main/java/org/akaza/openclinica/control/admin/CreateXformCRFVersionServlet.java]: I need that url for  debuging my local Form Manager application.

File: core/src/main/resources/properties/item_dao.xml (hunk 1 above)
Comment (jkeremian) [inline core/src/main/resources/properties/item_dao.xml]: Due to recent changes to crf_version and  form_layout , where we use the same crf_version against all the from_layouts within CRF, caused the versioning_map table to have multiple [item and crf_version combination] records for each form_layout within CRF.  For this reason, the page was loading duplicate records. By adding distinct to the sql, I'm able to get unique records.

**Comment by yogi-oc** [comment]
great responses to my comments.

**Pull request merged**